lib/libc/stdlib/malloc/mrs/mrs.3

```diff
@@ -208,6 +230,31 @@ permission.
 This capability will not be revoked and thus can be stored in the quarantine
 structure.
 This interface must not be exported outside the implementation.
+.Sh EXAMPLES
+If a program linked with
+.Nm
+allocator wishes to use revocation even if the system-wide default is to
+disable it, the programmer can add the following global decleration:
+.Pp
+.Dl int malloc_quarantining = MQ_ENABLED;
```

Comment:
Related to the discussion on Slack, I think `malloc_revoking` and `MR_*` for the constants might be the better names to use here considering the query function is `malloc_is_revoking`?  I don't know that you need the `_preference` suffix you suggested, I think just `malloc_revoking` is fine.

Reply:
I'd say `malloc_revoke = MQ_ENABLED` as that feels more command-y, whilst malloc_is_revoking is more verbal and gives you the actual current state. Also see cherirevoke as the elfctl name.

Reply:
Or malloc_revocation = to not collide with _that_ function name...